packages/ui/components/ui/dutch-auction-component/expanded-details/index.tsx

```diff
@@ -39,8 +39,8 @@ export const ExpandedDetails = ({
     fullSyncHeight >= description.startHeight &&
     fullSyncHeight <= description.endHeight;
 
-  const inputExponent = getDisplayDenomExponent(inputMetadata);
-  const outputExponent = getDisplayDenomExponent(outputMetadata);
+  const inputExponent = getDisplayDenomExponent.optional()(inputMetadata);
+  const outputExponent = getDisplayDenomExponent.optional()(outputMetadata);
```

Comment:
Bug fix?

Reply:
Yep — weirdly, some asset metadata was missing in testnet that I'd had in my wallet.